README.md

```diff
@@ -1,4 +1,8 @@
-# MAXSwerve Java Template
+# MAXSwerve Java Template v1.1
+
+See [CHANGELOG.md](CHANGELOG.md) for info about previous versions.
```

Comment:
```suggestion
See [the online changelog](https://github.com/REVrobotics/MAXSwerve-Java-Template/blob/main/CHANGELOG.md) for information about updates to the template that may have been released since you created your project.
```